### DISCUSSION TITLE
codespace

### DISCUSSION THREAD

**Discussion by anaadilj** [community/community#31233] [Codespaces]
I am facing an issue with code space. its being connected only showing stopping codespace. please solve this issue

**Comment by safwansamsudeen** [comment]
Hi,
GitHub has solved the issue - it happened to me, too! Could you check if it works for you now? It does for me. 
<img width="1321" alt="image" src="https://user-images.githubusercontent.com/62411302/187323102-189dcf82-8c6e-4c35-8304-de4c178fda0e.png">

**Comment by anaadilj** [comment]
yes done. but now in the terminal window when I am trying to run a program
it shows lets say scrabble is a directory. what does this means

**Reply by safwansamsudeen** [reply]
I’m afraid I don’t know… could you attach a picture, maybe?

**Comment by anaadilj** [comment]
screenshot is attached below

**Reply by safwansamsudeen** [reply]
There's no screenshot.
